addons/web/static/src/views/fields/field.js

```diff
@@ -67,7 +67,7 @@ fieldRegistry.addValidation({
     },
     useSubView: { type: Boolean, optional: true },
     label: { type: [String, { value: false }], optional: true },
-    listViewWidth: { type: [Number, [Number, Number], Function], optional: true },
+    listViewWidth: { type: [Number, { type: Array, validate: (array) => array.length === 2 }, Function], optional: true },
```

Comment:
Yep, that's my fault, `[Number, Number]` means number or number, not pair of numbers.
```suggestion
    listViewWidth: { type: [Number, { type: Array, elements: Number, validate: (array) => array.length === 2 }, Function], optional: true },
```